networkx/algorithms/traversal/depth_first_search.py

```diff
@@ -128,14 +147,15 @@ def dfs_predecessors(G, source=None):
     -----
     Based on http://www.ics.uci.edu/~eppstein/PADS/DFS.py
     by D. Eppstein, July 2004.
-
+    https://en.wikipedia.org/wiki/Depth-limited_search
     If a source is not specified then a source is chosen arbitrarily and
     repeatedly until all components in the graph are searched.
     """
-    return dict((t,s) for s,t in dfs_edges(G,source=source))
+    return dict((t, s) for s, t in dfs_edges(G, source=source,
```

Comment:
Since you are already changing this line, might as well use a dictionary comprehension instead (see #1640):

    return {t: s for s, t in dfs_edges(G, source, depth_limit)}